data/moves.js

```diff
@@ -1679,7 +1679,9 @@ let BattleMovedex = {
 					return;
 				}
 				if (source.volatiles['lockon'] && target === source.volatiles['lockon'].source) return;
-				return false;
+				if (!move.spreadHit) this.attrLastMove('[miss]');
+				this.add('-miss', source, target);
+				return null;
```

Comment:
I always get concerned when this sort of thing gets less DRY. What's changing here and why? Was it not a miss message in the past?

Reply:
Well, I hate to concern you more, but I don't exactly remember why I did this. I've been working on this for a while, and I've honestly forgotten what some stuff does. Looking at what's written now, I _think_ this might not be necesary?

Reply:
:[

Reply:
You know I'm not going to merge this unless you figure it out.

And ideally write a better solution.

Reply:
Yeah, turned out this was unnecessary.